kubernetes-manifests/frontend.yaml

```diff
@@ -88,6 +88,8 @@ spec:
           #   value: "true"
           # - name: FRONTEND_MESSAGE
           #   value: "Replace this with a message you want to display on all pages."
+          - name: PACKAGING_SERVICE_URL
+            value: "" # This value should look like "http://123.456.789/"
```

Comment:
Just that comment might be confusing to users. "What is the packaging service url? Do I need this? Where do I find more information about it?"

Reply:
I've updated the comment here, and also commented out the YAML fields.
```
          # As part of a Google Cloud demo, you can run an optional microservice called the "packaging service".
          # - name: PACKAGING_SERVICE_URL
          #   value: "" # This value would look like "http://123.456.789/"
```


Unfort, there is no public documentation for this demo yet (that we can reference here).